app/directives/forms/edit/reservation.js

```diff
@@ -338,6 +338,15 @@ define(['app', 'lodash',
                             });
                         } //init
 
+                        //============================================================
+                        //
+                        //============================================================
+                        $scope.setVideoUrl= function(doc) {
+                            if(doc.video)
+                              doc.videoUrl=$scope.room.videoUrl
```

Comment:
maybe this is better `doc.video = {url : ''}`

Reply:
Why?  I'm not sure I get it.

Reply:
@blaisefonseca I have the same idea last week. but I thibk we will go the the two fields and discuss this way of doing thing later

Reply:
@randyhoulahan The idea behind blaise comment is to avoid breaking concept. everthing related to video is in the `video` prop. if `video` is defined then it nean true and all the other value ar in the video objetc. it to avoid repeating videoUrl, videoTitle, videoXYZ etc. Since mongo is a document ans not a flat table like SQL.

Reply:
Yes, I see now @blaisefonseca.  Yes that was my initial go to as well.  @sbilodeau and I talked about it quickly to do this instead.  Although I did not he his need to separate yet, I trusted as I was in hurry.

Reply:
@sbilodeau Should I make it like this then?  Or keep it as is?

Reply:
@randyhoulahan I think it's better to be consistent and keep the same pattern that you already have in the reservation object.